plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/AbstractProjectTest.java

```diff
@@ -30,7 +30,7 @@
 /**
  * Common infrastructure for workbench-based tests that create a single project.
  */
-public class AbstractProjectTests {
+public class AbstractProjectTest {
```

Comment:
I did another test to verify this: as long as the inheriting class ends with `Test`, the name of the superclass doesn't matter.

Anyway, I renamed this to `BaseProjectTest`. (Not `Abstract`.)